lisa/sut_orchestrator/azure/common.py

```diff
@@ -46,9 +46,11 @@ class EnvironmentContext:
 class NodeContext:
     resource_group_name: str = ""
     vm_name: str = ""
+    vm_size = ""
```

Comment:
I double checked the code, they should come from node.capability.get_extended_runbook, not from context. It avoids duplicating those values.